README.md

```diff
@@ -69,59 +85,67 @@ stack install
 
 To type the calculation above:
 
-* type "**`1+s`**"
-* at this point we apply the addition operator to the number 1 and are picking the other argument to the addition. we are offered different options which contain the string "s" in their names
-* pick the option we desire ("sqrt _") by filtering the option by typing and navigating to it with the cursor keys, and then pressing **space** (as of writing this, it is the first option offered, partly due to a currently minimal library of available functions).
-* type "**`5`**" and press **space**
-* select the whole expression by pressing **shift+left** twice
-* type "**`/2`**" and press **space**
+* Type "**`1+s`**" at the prompt (`⋙`). Notice we have chosen "`1`" for the addition's left argument. However, we have only begun to type the second argument: it starts with an "s". Lamdu knows we have finalized the left argument because we have moved on from it, indicated by the `+`. But we have done nothing to indicate that just `s` is the second argument. To help us finalize the right argument, Lamdu has presented a menu of type-appropriate choices containing "s" in their names &ndash; "containing", not just "starting with". This menu updates as we type.
+* Next, we will flesh out the "s" into a "sqrt". As of September 2017, "sqrt" should already be selected in the nearby menu, because it is alphabetically the first function in the library to contain an "s" in its name and to output a number. However your menu, take the path of fewest keystrokes: continuing to type the function's name reduces the menu options to just those that match. Cursor keys allow you to select from the menu. Hit **space** to chose your selected menu option.
+* Type "**`5`**" and press **space**.
+* Select the whole expression by pressing **shift+left** twice.
+* Type "**`/2`**" and press **space**. Notice that Lamdu just automatically inserted the parentheses.
 
-Directly under each expression in the code we see annotations of its evaluation results. Some annotations are more far apart from the expression's code due to its subexpression's annotation appearing between them.
+Lamdu displays the evaluation of each expression, whether the whole or a subexpression. Such an automatic display is called an "annotation." The annotation of an expression appears below that of any child expression. For example, the evaluation of `(1 + sqrt 5) / 2` appears below that of its child expression, `(1 + sqrt 5)`. The former is `1.61...` and the latter is `3.23...`.
 
-So as to not expand the expression, some annotations are shrunk. In this case `sqrt 5`'s annotation is shrunk. To see it in normal size - navigate to this expression, by going to the `sqrt` or `5` and pressing **shift+left**.
+To keep the expression's font size consistent, some annotations are shrunk, like that of the `sqrt 5` above, which is `2.23...`. To see this in normal size, navigate to the expression by going to the `sqrt`, or to the `5`, and press **shift+left**.
```

Comment:
The "expansion" we had before shrinking annotations wasn't by changing the font size but rather by adding a padding around it so the annotation could fit. So the mention of font size is somewhat misleading.

Reply:
Sorry. That should be fixed, then. It's just that "expanding an expression" usually means something mathy. For example, expanding "( 1 + sqrt 5) / 2" could be " 0.5 + 0.5 * sqrt 5".